### ISSUE TITLE
Accessibility Bug: When the ‘button’ in the shell bar is accessed, reading is made as ‘Popover button Button’.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3146]
Fundamental core component: "Product switch"

Description: When the ‘button’ in the shell bar is accessed, reading is made as ‘Popover button Button’.

Expected:  The extra ‘button’ announcement should be removed when the button from the shell bar region is accessed.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91651554-c72ace80-eaab-11ea-9059-5dac7311b2a0.png)
